src/autoloads/characters.gd

```diff
@@ -69,6 +69,17 @@ func getCharacterNode(id: int) -> Node:
 		return null
 	return _characterNodes[id]
 
+func removeCharacterNode(id: int) -> void:
+	# if there is no character node corresponding to this network id
+	if not id in _characterNodes:
+		# throw an error
+		printerr("Trying to get a nonexistant character node with network id ", id)
+		# crash the game (if running in debug mode) to assist with debugging
+		assert(false, "Should be unreachable")
+		# if running in release mode, return
+		return
+	_characterNodes.erase(id)
```

Comment:
is it enough to erase it from the dictionary? don't we have to "destroy" the instance somehow? I'm not too familiar with how garbage collection in Godot works.

Reply:
I think it's okay.
This is all the docs say about erase()


    bool erase ( Variant key )

Erase a dictionary key/value pair by key. Returns true if the given key was present in the dictionary, false otherwise.

Note: Don't erase elements while iterating over the dictionary. You can iterate over the keys array instead.